course3/homework-string/test.cc

```diff
@@ -53,11 +93,11 @@ int main()
 {
 	// pre-test - of this _doesn't_ crash your program, you faield to compile it in debug mode (g++ -g)
 	// if it crashes, it's a good sign. Remove the line and proceed.
-	assert(false);
+	//assert(false);
 	
 	// test fior std::string - works
 	std::cout << "Testing std::string..." << std::endl;
-	string_test<std::string>();
+	//string_test<std::string>();
```

Comment:
Didn't intend to leave this commented out, I was testing with std::cout output to help figure out what methods got invoked and disabled this to reduce the amount of output.